CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# v3.5.23
+
+- This change significantly reduces the weight of buffers latent on the heap by
+  using a shared 64KB buffer instead of one intermediate buffer for each frame.
+- Removing a connection from a peer now properly cleans up event listeners on
+  the connection.
+- Pending request counts are now tracked incrementally.
+- Adds Peer#toString and Peer#inspect
+- Fixes Peer#connectTo such that it returns a Connection.
+- Adds a missing TChannel#setWriteBufferMode(mode) method to retain
```

Comment:
This is a hyperbahn implementation detail; feel free to drop it from this changelog.